opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/Configuration.java

```diff
@@ -315,6 +316,15 @@
         ON, OFF, DIRBASED, UIONLY
     }
 
+    public enum supportedMeterRegistryType {
```

Comment:
I don't remember if I have ever seen a type with lowercase first letter

Reply:
Anyway, I think MetricType is enough ... the "supported" part is kinda the purpose of the enum.

Reply:
I will drop the 'supported' prefix. MetricType would be misleading - it should rather be MeterType however that would be imprecise since meter type is something like Timer, Gauge etc.